src/components/Questions/QuestionMultiple.vue

```diff
@@ -160,6 +187,14 @@ export default {
 		titleId() {
 			return `q${this.$attrs.index}_title`
 		},
+
+		isShowOtherAnswer() {
```

Comment:
```suggestion
		hasOtherAnswer() {
```

Better wording for method